src/webapp/pages/form-page/disease-outbreak-event/utils/diseaseOutbreakEventFormMapper.ts

```diff
@@ -0,0 +1,806 @@
+import {
+    DiseaseOutbreakEventBaseAttrs,
+    IncidentStatusType,
+} from "../../../../../domain/entities/disease-outbreak-event/DiseaseOutbreakEvent";
+import { DiseaseOutbreakEventWithOptions } from "../../../../../domain/entities/disease-outbreak-event/DiseaseOutbreakEventWithOptions";
+import {
+    FormFieldState,
+    FormState,
+    getAllFieldsFromSections,
+    getDateFieldValue,
+    getMultipleOptionsFieldValue,
+    getBooleanFieldValue,
+    getStringFieldValue,
+} from "../../../../components/form/FormState";
+import { Option } from "../../../../../domain/entities/Ref";
+import { getFieldIdFromIdsDictionary } from "../../../../components/form/FormState";
+import { UserOption } from "../../../../components/user-selector/UserSelector";
+import { Option as PresentationOption } from "../../../../components/utils/option";
+import { getHazardTypeValue } from "../../../../../data/repositories/consts/DiseaseOutbreakConstants";
+
+export const diseaseOutbreakEventFieldIds = {
```

Comment:
why are we creating this map? Can we not use the keys as ids ?

Reply:
I just wanted to centralise the ids at one point so that I can use them in both the mapToFormState and the mapToEntity. Any ideas?

Reply:
my query was why the id cannot be the same as the key? We are setting it ourselves right? For example :  emergedDate: "emerged_date" - why not use "emergedDate" as id? I hope I am understanding correctly.